src/pydantic_cereal/main.py

```diff
@@ -258,11 +296,18 @@ def _pop_context(self, ctx: CerealContext) -> None:
         self._context_stack.pop()
 
     @property
-    def workdir(self) -> UPath:
+    def fs(self) -> AbstractFileSystem:
+        """Current filesystem."""
+        if self.active_context is None:
+            raise CerealContextError("No context is active - no current filesystem.")
+        return self.active_context.fs
+
+    @property
+    def workdir(self) -> str:
         """Working directory."""
         if self.active_context is None:
             raise CerealContextError("No context is active - no working directory.")
-        return self.active_context.workdir
+        return self.active_context.target_path
```

Comment:
Not sure whether we should rename it from `workdir` to `target_path` here, or the other way in the context, but they probably should have the same name.